### PR TITLE
FW-2292 Ask for CSAT from bot reply [Chat-widget]

### PR DESCRIPTION
**Pull request by Ojashwa**
### What do you want to achieve?
- Ask for CSAT from bot reply.

### PR Checklist

- [x] I have tested it locally and all functionalities are working fine.
- [ ] I have compared it with mocks and all design elements are the same.
- [x] I have tested it in IE Browser.

### How was the code tested?

---
https://user-images.githubusercontent.com/22856546/126479139-41dcc31c-5f35-49d0-ba7e-4980d41bcbe5.mov

```
{
    "platform": "kommunicate",
    "message": "Hope we answered all your queries. Please reach out to us in case of any more queries.",
    "metadata": {
        "actionRequest": "askCSAT"
    }
}
```

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

**Review by ChannuBS — APPROVED** [review]

File: webplugin/js/app/mck-sidebox-1.0.js [hunk 1]
```diff
@@ -15919,6 +15919,14 @@ var userOverride = {
                             '#mck-sidebox-content'
                         );
                         var tabId = $mck_message_inner.data('mck-id');
+                        if (resp.message.metadata.actionRequest == 'askCSAT') {
```
Comment (Prochnost55) [inline webplugin/js/app/mck-sidebox-1.0.js:15922]: what happens if metadata is not defined?

Comment (devashishmamgain) [inline webplugin/js/app/mck-sidebox-1.0.js:15922]: if we ever change something then and miss to change at any one of the place, then bugs will come,  call the same function code from both places

document.getElementById('km-csat-trigger').onclick = function (
                    e
                ) {
                    e.preventDefault();
                    KommunicateUI.triggerCSAT();
                    kommunicateCommons.modifyClassList(
                        { id: ['mck-csat-close'] },
                        'vis',
                        'n-vis'
                    );
                };

Comment (Ojashwa) [reply webplugin/js/app/mck-sidebox-1.0.js]: Here, In `kommunicateCommons.modifyClassList()` we are adding `vis ` class in my case we are adding `n-vis` class.
There is one more occurence for what you pointing for have fixed that.

**Review by devashishmamgain — CHANGES_REQUESTED** [review]
There must be one more place
document.getElementById('km-csat-trigger').onclick

**Comment by Ojashwa** [comment]
> 
> 
> There must be one more place
> document.getElementById('km-csat-trigger').onclick
- Found this occurance with the suggested text.
----
image
---
- Here class name is different.
image

**Review by devashishmamgain — APPROVED** [review]

**Review by Prochnost55 — APPROVED** [review]

**Review by ChannuBS — APPROVED** [review]

**Pull request merged**